datacapturing/src/main/java/de/cyface/datacapturing/DataCapturingService.java

```diff
@@ -982,13 +1014,54 @@ public void onBindingDied(final @NonNull ComponentName name) {
         @Override
         public void handleMessage(final @NonNull Message msg) {
             Log.v(TAG, String.format("Service facade received message: %d", msg.what));
+            final Bundle parcel;
+            parcel = msg.getData();
+            parcel.setClassLoader(getClass().getClassLoader());
+
+            // The Broadcasts for the ShutdownFinishedHandler need to be independent from the number of registered
+            // CapturingListeners
+            switch (msg.what) {
+                case MessageCodes.SERVICE_STOPPED:
+                    // Due to the <code>DataCapturingBackgroundService#informCaller()</code> interface
```

Comment:
It is unclear what you mean by that.

Reply:
I removed it, I think it's clear why the parcel Bundle contains another Bundle (data) because this can contain multiple Objects and we wanted to have an equal interface for all messages.